### PR TITLE
Completed prep work for quarkus_openshift tests

### PR DESCRIPTION
**Pull request by SAY-droid427**
Completed prep work for quarkus_openshift tests
- Added README.md for quarkus_openshift tests
- Added test.properties

Close https://github.com/AdoptOpenJDK/openjdk-tests/issues/2442

### CODE REVIEW

File: external/quarkus_openshift/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+7. export required environment variables, BUILD_LIST and EXTRA_DOCKER_ARGS (`export BUILD_LIST=external/quarkus_openshift` and `export EXTRA_DOCKER_ARGS="-v $TEST_JDK_HOME:/opt/java/openjdk"`
+
+8. `make compile` (This fetches test material and compiles it, based on build.xml files in the test directories)
+
+9.  `make _quarkus_openshift_test` (When you defined BUILD_LIST to point to a directory in openjdk-tests/external, then this is a testCaseName from the playlist.xml file within the directory you chose)
+
+When [running these from the command-line](https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/doc/userGuide.md#local-testing-via-make-targets-on-the-commandline), these tests are grouped under a make target called 'external', so 'make external' would run the entire set of tests found in the openjdk-tests/external directory. This is unadvisable! Limit what you compile and run, BUILD_LIST=external/`<someSubDirectory>`, and TARGET=`<testCaseNameFromSubdirPlaylist>`.
+
+These tests run regularly and results can be found in [TRSS Third Party Application view](https://trss.adoptopenjdk.net/ThirdPartyAppVie).
```
Comment (smlambert) [inline external/quarkus_openshift/README.md:31]: missing 'w' from the end of ThirdPartyAppView

**Review by smlambert — COMMENTED** [review]
hi @SAY-droid427 - some minor changes requested, otherwise, looking pretty good!

File: external/quarkus_openshift/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# External quarkus_openshift Tests
+
+Third Party container tests help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue #172 lists the applications that we have initially targeted to best exercise the AdoptOpenJDK binaries. For each application, we choose to run a selection of their functional tests.
```
[the remark this answers is not in the document]
Comment (SAY-droid427) [reply external/quarkus_openshift/README.md]: Done

File: external/quarkus_openshift/README.md (hunk 1 above)
Comment (SAY-droid427) [reply external/quarkus_openshift/README.md]: Corrected

File: external/quarkus_openshift/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+# External quarkus_openshift Tests
+
+Third Party container tests help verify that the Adoptium binaries are good by running a variety of Java applications inside of Docker containers. AdoptOpenJDK/openjdk-tests/Issue [#172](https://github.com/AdoptOpenJDK/openjdk-tests/issues/172) lists the applications that we have initially targeted to best exercise the Adoptium binaries. For each application, we choose to run a selection of their functional tests. Quarkus Openshift tests are pulled from the [quarkus-openshift-test-suite](https://github.com/quarkus-qe/quarkus-openshift-test-suite.git) repository. These tests require an Openshift cluster environment to run.
```
Comment (sophia-guo) [inline external/quarkus_openshift/README.md:3]: Please change the first part of the sentence to be:
Quarkus_openshift tests are part of the external third-party application tests that help verify that the Adoptium binaries are good...

**Comment by sophia-guo** [comment]
Please also sign off your commit and update your comment https://github.com/AdoptOpenJDK/openjdk-tests/pull/2454#issue-608013711 with Close #2442 , which will automatically close the issue #2242 when this PR is merged.

**Review by sophia-guo — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]
👍

**Pull request merged**